styles/globals.scss

```diff
@@ -118,3 +118,43 @@ body {
 .cursor-pointer {
     cursor: pointer;
 }
+#notification {
+    animation: slide-in 1s ease-in-out normal;
+    overflow: hidden;
+}
+
+@keyframes slide-in {
+    0% {
+        opacity: 0;
+        transform: translateY(-100%);
+    }
+    100% {
+        opacity: 1;
+        transform: translateY(0);
+    }
+}
+
+#notification::after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 0.2rem;
+    position: absolute;
+    bottom: 0;
+    margin-bottom:-0.05rem;
+    opacity: 0.2;
+    left: 0;
+    background-color: rgba(0, 0, 0);
+    z-index: 1;
+    transform-origin: left;
+    animation: shrink var(--data-timeout) ease-in-out forwards;
+}
```

Comment:
This can be moved inside `#notification` 

```css
#notification {
    animation: slide-in 1s ease-in-out normal;
    overflow: hidden;
    
    &::after {
        content: '';
        display: block;
        width: 100%;
        height: 0.2rem;
        position: absolute;
        bottom: 0;
        margin-bottom:-0.05rem;
        opacity: 0.2;
        left: 0;
        background-color: rgba(0, 0, 0);
        z-index: 1;
        transform-origin: left;
        animation: shrink var(--data-timeout) ease-in-out forwards;
    }
}
```